docs/packaging.rst

```diff
@@ -2,5 +2,86 @@
 Packaging Guide
 ===============
 
-Documentation can be found at
-https://github.com/certbot/certbot/wiki/Packaging.
+Releases
+========
+
+We release packages and upload them to PyPI (wheels and source tarballs).
+
+- https://pypi.python.org/pypi/acme
+- https://pypi.python.org/pypi/certbot
+- https://pypi.python.org/pypi/certbot-apache
+- https://pypi.python.org/pypi/certbot-nginx
+
+The following scripts are used in the process:
+
+- https://github.com/letsencrypt/letsencrypt/blob/master/tools/release.sh
+
+We currently version with the following scheme:
+
+- ``0.1.0``
+- ``0.2.0dev`` for developement in ``master``
+- ``0.2.0`` (only temporarily in ``master``)
+- ...
+
+Notes for package maintainers
+=============================
+
+0. Please use our releases, not ``master``!
+
+1. Do not package ``certbot-compatibility-test`` or ``letshelp-certbot`` - it's only used internally.
+
+2. If you'd like to include automated renewal in your package ``certbot renew -q`` should be added to crontab.
```

Comment:
With systemd taking over, should we say "should be added to a crontab or systemd timer"?